### PR TITLE
Update project name

### LINKED ISSUE #1: [Feature]: Open Source skills development
### Description

Develop your Open Source skills 🎉 

### Expected behaviour

Here is what you have to do :

- [x] Create a project from [PoC's project template](https://github.com/PoCInnovation/open-source-project-template) and play with :
  - [x] Milestones
  - [x] Issues
  - [x] Pull Requests
  - [x] Releases
  - [x] Labels
  - [x] GitHub project (beta version)

- [x] Update the [README.MD](https://github.com/PoCInnovation/MindGame/blob/master/README.md) file and the [contributing guide](https://github.com/PoCInnovation/open-source-project-template/blob/main/.github/getting-started.md)

### Environment

/

### Proposed solution

/

### PR DESCRIPTION
**Pull request by Baragouin**
# Description

Project name in `CONTRIBUTING.MD` needed to be changed

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have tested this code
- [ ] I have added / updated tests
- [x] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-# Contributing to [PROJECT'S NAME]
+# Contributing to MindGame
 
 ## Create an issue
 
 - If you've encountered a bug, open a [Bug Report](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=bug_report.md&title=).
-- If you want [PROJECT'S NAME] to have a new fonctionality, open a [Feature Request](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=feature_request.md&title=).
+- If you want [MindGame] to have a new fonctionality, open a [Feature Request](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=feature_request.md&title=).
```
Comment (mvallenet) [inline CONTRIBUTING.md]: Delete the brackets & change the $REPOSITORY variable

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 ## Resolve an issue
 
 Select an [issue](https://github.com/PoCInnovation/$REPOSITORY/issues) that you want to resolve.
 
 The recommended workflow is to fork this repository and open pull requests from your fork.
 
-### 1. Fork, clone & configure [PROJECT'S NAME] upstream
+### 1. Fork, clone & configure [MindGame] upstream
```
Comment (mvallenet) [inline CONTRIBUTING.md]: delete brackets

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 ## Create an issue
 
 - If you've encountered a bug, open a [Bug Report](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=bug_report.md&title=).
-- If you want [PROJECT'S NAME] to have a new fonctionality, open a [Feature Request](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=feature_request.md&title=).
+- If you want [MindGame] to have a new fonctionality, open a [Feature Request](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=feature_request.md&title=).
 
 ## Resolve an issue
 
 Select an [issue](https://github.com/PoCInnovation/$REPOSITORY/issues) that you want to resolve.
```
Comment (mvallenet) [inline CONTRIBUTING.md]: change the $REPOSITORY variable per the name of the project

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
-# Contributing to [PROJECT'S NAME]
+# Contributing to MindGame
 
 ## Create an issue
 
 - If you've encountered a bug, open a [Bug Report](https://github.com/PoCInnovation/$REPOSITORY/issues/new?assignees=&labels=&template=bug_report.md&title=).
```
Comment (mvallenet) [inline CONTRIBUTING.md]: change $REPOSITORY variable

**Review by mvallenet — COMMENTED** [review]
Reread the contribution guide and make sure you have replaced all $REPOSITORY variables and the [PROJECT'S NAME].

**Comment by Baragouin** [comment]
I'm changing this, I thought they were GITHUB variables 🤔

**Review by mvallenet — APPROVED** [review]

**Pull request merged**